PhysicalBadging.postman_collection.json

```diff
@@ -0,0 +1,90 @@
+{
+	"info": {
+		"_postman_id": "5a0ec7aa-7191-4950-b817-0ec1468f54d9",
+		"name": "PhysicalBadging",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Save Signals",
+			"request": {
+				"auth": {
+					"type": "bearer",
+					"bearer": [
+						{
+							"key": "token",
+							"value": "",
+							"type": "string"
+						}
+					]
+				},
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "[\n\t{\n\t\t\"AssetId\":\"Mid-Sec-7\",\n\t\t\"AssetName\":\"Main Building 1st Floor Mid Section\",\n\t\t\"AccessStatus\":\"Failed\",\n\t\t\"EventTime\":\"2019-07-04T01:57:49\",\n\t\t\"UserId\":\"tifc@a830edad9050849EQTPWBJZXODQ.onmicrosoft.com\",\n\t\t\"SourceSystem\":\"Lenel\"\n\t},\n\t{\n\t\t\"AssetId\":\"Mid-Sec-7\",\n\t\t\"AssetName\":\"Main Building 1st Floor Mid Section\",\n\t\t\"AccessStatus\":\"Success\",\n\t\t\"EventTime\":\"2019-07-04T02:57:49\",\n\t\t\"UserId\":\"tifc@a830edad9050849EQTPWBJZXODQ.onmicrosoft.com\",\n\t\t\"SourceSystem\":\"Lenel\"\n\t}\n]",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{host}}/api/signals/physicalbadging?jobid={{jobId}}",
+					"host": [
+						"{{host}}"
+					],
+					"path": [
+						"api",
+						"signals",
+						"physicalbadging"
+					],
+					"query": [
+						{
+							"key": "jobid",
+							"value": "{{jobId}}"
+						}
+					]
+				}
+			},
+			"response": []
+		}
+	],
+	"event": [
+		{
+			"listen": "prerequest",
+			"script": {
+				"id": "6f3e901a-37a7-4fab-8188-65f7b8afa663",
+				"type": "text/javascript",
+				"exec": [
+					""
```

Comment:
if this is empty, does the postman still export it ?

Reply:
yes